corehq/apps/locations/management/commands/migrate_admin_status.py

```diff
@@ -0,0 +1,50 @@
+# One-off migration from 2016-04-04
+from optparse import make_option
+from time import sleep
+from django.core.management.base import BaseCommand
+from corehq.apps.locations.models import LocationType, SQLLocation
+from corehq.apps.es import DomainES
+from corehq.util.log import with_progress_bar
+
+
+def get_affected_location_types():
+    commtrack_domains = (DomainES()
+                         .commtrack_domains()
+                         .values_list('name', flat=True))
+    return (LocationType.objects
+            .exclude(domain__in=commtrack_domains)
+            .filter(administrative=False))
+
+
+def show_info():
+    location_types = get_affected_location_types()
+    num_locations = SQLLocation.objects.filter(location_type__in=location_types).count()
+    print ("There are {domains} domains, {loc_types} loc types, and "
+           "{locations} locations affected").format(
+        domains=location_types.distinct('domain').count(),
+        loc_types=location_types.count(),
+        locations=num_locations,
+    )
+
+
+def run_migration():
+    for location_type in with_progress_bar(get_affected_location_types()):
+        if not location_type.administrative:
+            location_type.administrative = True
+            location_type.save()
+            sleep(1)
```

Comment:
Oh awesome, this is much less prevalent that I'd expected
> There are 24 domains, 70 loc types, and 1100 locations affected

The task finished without error